src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -4,16 +4,25 @@
 using LaunchDarkly.Logging;
 using LaunchDarkly.Sdk.Server;
 using LaunchDarkly.Sdk.Server.Integrations;
+using LaunchDarkly.Sdk.Server.Interfaces;
 
 namespace Bit.Core.Services;
 
-public class LaunchDarklyFeatureService : IFeatureService, IDisposable
+public class LaunchDarklyFeatureService : IFeatureService
 {
-    private readonly LdClient _client;
+    private readonly ILdClient _client;
+    private readonly ICurrentContext _currentContext;
     private const string _anonymousUser = "25a15cac-58cf-4ac0-ad0f-b17c4bd92294";
 
     public LaunchDarklyFeatureService(
-        IGlobalSettings globalSettings)
+        ILdClient client,
+        ICurrentContext currentContext)
+    {
+        _client = client;
+        _currentContext = currentContext;
+    }
+
+    public static Configuration GetConfiguredClient(GlobalSettings globalSettings)
```

Comment:
I would normally saw that this type of thing should be in the service factory but since this is more code than would normally be in a factory I do like that it stays here.